multiversx_sdk_cli/cli_shared.py

```diff
@@ -274,8 +275,16 @@ def send_or_simulate(tx: ITransaction, args: Any, dump_output: bool = True) -> C
             simulation = Simulator(proxy).run(tx)
             output_builder.set_simulation_results(simulation)
     finally:
+        output_transaction = output_builder.build()
+
         if dump_output:
-            utils.dump_out_json(output_builder.build(), outfile=outfile)
+            utils.dump_out_json(output_transaction, outfile=outfile)
+
+        if send_only:
+            log_explorer_transaction(
```

Comment:
In a future PR, we have to adjust the implementation of `log_explorer_transaction` to take `config` items (not existing yet) into consideration. E.g. some might want to use the `devnet2` URL for chain ID `D`.

https://github.com/multiversx/mx-sdk-py-cli/blob/main/multiversx_sdk_cli/utils.py#L246

Reply:
Let's create a GitHub issue on this matter.

Reply:
I've thought about this, but since in about one month there will be no `devnet2` I decided to leave it as it is.